package.json

```diff
@@ -18,9 +18,9 @@
     "prod": "pnpm docker:compose:prod up -d --build",
     "dev:update": "pnpm docker:compose up -d --build && pnpm wallet-frontend dev",
     "dev:backend:legacy": "pnpm wallet-backend build && pnpm dev && nodemon --watch packages/wallet-backend -e ts --exec 'pnpm docker:compose:legacy up -d --no-deps --build wallet-backend'",
+    "dev:backend:update": "pnpm wallet-backend-update build && pnpm dev:update && nodemon --watch packages/wallet-backend-update -e ts --exec 'pnpm docker:compose up -d --no-deps --build wallet-backend-update'",
     "docker:compose": "docker compose -f ./docker/dev/docker-compose.yml",
     "docker:down": "pnpm docker:compose down",
-    "docker:compose:legacy": "docker compose -f ./docker/dev/docker-compose-legacy.yml",
```

Comment:
If this is missing, then pnpm dev will give error:
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command failed with ENOENT: docker:compose:legacy up -d --build
spawn docker:compose:legacy ENOENT
Command "docker:compose:legacy" not found. Did you mean "pnpm run docker:down:legacy"?